mcc/checker.wsgi

```diff
@@ -10,10 +10,11 @@ sys.path.append(os.path.dirname(__file__))
 
 
 def application(environ, start_response):
-	os.environ['MaxFileSize'] = environ.get('MaxFileSize', '')
-	os.environ['HomepageURL'] = environ.get('HomepageURL', '')
-	os.environ['TempFileLocation'] = environ.get('TempFileLocation', '')
-	os.environ['Venue'] = environ.get('Venue', 'OPS')
-	from web.server import app as _application
+    os.environ['MaxFileSize'] = environ.get('MaxFileSize', '')
+    os.environ['HomepageURL'] = environ.get('HomepageURL', '')
+    os.environ['TempFileLocation'] = environ.get('TempFileLocation', '')
+    os.environ['Venue'] = environ.get('Venue', 'OPS')
+    os.environ['CF_STANDARD_NAME_TABLE'] = environ.get('CF_STANDARD_NAME_TABLE', '')
```

Comment:
Do we want these to hard fail with a KeyError if they're not defined? Seems like it could lead to odd downstream behaviors if there's an invalid config here

Reply:
We could certainly use some more sensible defaults for the first three.

For the standard name table, the `compliance-checker` code will only attempt to use the file referenced by this envvar if it actually exists on disk, otherwise it falls back the table bundled with the repo: https://github.com/ioos/compliance-checker/blob/main/compliance_checker/cf/util.py#L217-L218

But we should at least perform the same existence check here and log a warning if we don't find the file. Otherwise the CF checker will silently fall back to the outdated version of the table.